configs/terminate_connect.yaml

```diff
@@ -57,8 +64,3 @@ static_resources:
               socket_address:
                 address: www.google.com
                 port_value: 443
-    transport_socket:
```

Comment:
I'm guessing you didn't mean to delete this?

/wait

Reply:
Hi Alyssa, Thanks for the corrected version,
I just tried it out and it works like a charm,
Kannan

Reply:
I did mean to delete it, but I pushed the wrong comments.

Apologies for the force push - I'd fixed the information locally and somehow failed to  update upstream before I sent this out.